compete.html

```diff
@@ -130,7 +130,7 @@ <h3>Building a robot</h3>
       <a name="signup"></a><h2>Want to compete?</h2>
       <div class="column m-8-12 m-offset-2-12 text-left">
         <p>
-          Entries for this year's competition are open until the 26th October 2018.
+          Entries for Student Robotics 2020 are open until around September 2019.
```

Comment:
Should we perhaps call this "Applications" to help further clarify that not everyone gets a place?
Just a thought.

Reply:
We should definitely do this!